es6-persistence/src/main/java/com/netflix/conductor/es6/config/ElasticSearchProperties.java

```diff
@@ -12,141 +12,192 @@
  */
 package com.netflix.conductor.es6.config;
```

Comment:
I think we can even change the package name to  just `es` or `elasticsearch` or `index` instead of `es6`.

Reply:
See comment for es6 condition above.